### ISSUE TITLE
Readme in English

### ISSUE THREAD

**Issue by samdark** [axiles89/yii2-coordinator#1]
Would be great to have readme in English.

**Comment by axiles89** [comment]
Да, добавлю конечно!

**Comment by samdark** [comment]
А можно ещё и анонс на http://yiifeed.com/ добавить? Расширение определённо полезное. Надо, чтобы о нём знали.

**Comment by axiles89** [comment]
Да, буду только признателен) В любое место можно и нужно добавлять!

**Comment by samdark** [comment]
Там можно авторизоваться через GitHub и предложить новости.

**Comment by axiles89** [comment]
Ага, сегодня сделаю. Сейчас не дома
